src/triage/cli.py

```diff
@@ -44,6 +46,21 @@ def __init__(self, parser):
             type=argparse.FileType('r'),
             help="database connection file",
         )
+        parser.add_argument(
+            '-s', '--setup',
+            help="file path to Python module to import before running the Experiment",
+        )
+
+    @cachedproperty
+    def setup(self):
+        setup_path = self.args.setup or os.path.exists('experiment.py')
+        if setup_path is not None:
+            logging.info(f"Loading configurations from {setup_path}")
+            spec = importlib.util.spec_from_file_location("triage_config", setup_path)
+            triage_config = importlib.util.module_from_spec(spec)
+            spec.loader.exec_module(triage_config)
+            logging.info(f"Configuration loaded")
+        return None
```

Comment:
There is some utility to this being a `cachedproperty` -- it'll only be run once -- but it's pretty weird. (And so you do weird things like `_ = self.root.setup`.)

This can just be a regular method.

Reply:
Ok